app/views/articles/feed.rss.builder

```diff
@@ -11,15 +11,11 @@ xml.rss version: "2.0" do
 
     @articles.each do |article|
       xml.item do
-        if article.title
-          xml.title article.title
-        else
-          xml.title ""
-        end
-        xml.author article.user.name
+        xml.title article.title.to_s
```

Comment:
I believe article.title is already a string.
```suggestion
        xml.title article.title
```

Reply:
Yeah. I put that there to force `nil` into being string because the previous if/else made me concerned. But I think that was a reaction to some prior situation where we had `nil` titled articles in. But on further inspection, there are 0 total articles with no title, so this can be removed.

Reply:
Ah, I see. I've never considered that use case before.